helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -325,7 +325,7 @@ splunk_hec/platform_traces:
   token: "${SPLUNK_PLATFORM_HEC_TOKEN}"
   index: {{ .Values.splunkPlatform.tracesIndex | quote }}
   source: {{ .Values.splunkPlatform.source | quote }}
-  max_connections: {{ .Values.splunkPlatform.maxConnections }}
+  max_idle_conns_per_host: {{ .Values.splunkPlatform.maxConnections }}
```

Comment:
`max_idle_conns` which is max idle connections across all host is [set to 100](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/splunkhecexporter/factory.go#L51) by default. We have `.Values.splunkPlatform.maxConnections = 200`. So we'll be setting per host limit greater than the total limit and I think effectively it will still be 100. Should we just set `max_idle_conns` = `max_idle_conns_per_host` = `.Values.splunkPlatform.maxConnections` instead? I assume the per `host` in http client is referring to the `host` part of the endpoint/uri and not per ip-address a host could resolve to.

Reply:
I think we can, yes. Wasn't sure. I have a separate PR for it now because this one keeps being overridden: https://github.com/signalfx/splunk-otel-collector-chart/pull/1034